File: client-app/src/models/payee.ts
export class Payee {
    id: string;
    payeeName: string;

    lastBudgetItem: string;

    constructor(id: string, title: string, lastBudgetItem: string = "") {
        this.id = id;
        this.payeeName = title;
        this.lastBudgetItem = lastBudgetItem;
    }
}

File: Domain/Models/Transactions/Payee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Enums.Transactions
{
    public class Payee
    {
        /// <summary>
        /// The ID of this <see cref="Payee"/>
        /// </summary>
        [Required]
        public Guid Id { get; set; }

        /// <summary>
        /// The title of the <seealso cref="Payee"/>
        /// </summary>
        public string? PayeeName { get; set; }
    }
}